### PR TITLE
Redirect InfernalRobotics-Sequencer from SpaceDock to GitHub

### PR DESCRIPTION
**Pull request by Darael**
SpaceDock hasn't been upgraded since IRS version 0.6.0, but the mod is
up to 1.0.0 on GitHub (and Sequencer versions each require the matching IR version).  I've checked with the mod authors and they've OK'd this update.

### CODE REVIEW

**Comment by politas** [comment]
Hi there! Thanks for the submission! One small thing: can you change "spec_version" back to "v1.4"? it's needed for "find" install directives.

File: NetKAN/InfernalRobotics-Sequencer.netkan [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "license": "GPL-3.0",
     "$kref": "#/ckan/github/MagicSmokeIndustries/IR-Sequencer",
     "$vref": "#/ckan/ksp-avc",
-    "spec_version": "1",
+    "spec_version": "1.4",
```
Comment (politas) [inline NetKAN/InfernalRobotics-Sequencer.netkan]: Needs to be "v1.4"

**Comment by Darael** [comment]
Sorry about that.  And also the delay.

**Comment by politas** [comment]
#rebuild

**Comment by DinCahill** [comment]
@Darael you need a comma on the ends of lines 7 and 12

**Comment by Darael** [comment]
Oh, for - 

I used to be better at this, honest.

Sorry.  This should improve matters.

**Comment by politas** [comment]
Looks good to me. Jenkins is happy, it seems to install all the files it ought to. Merging with thanks!

**Pull request merged**